integration_tests/test_commit_format.py

```diff
@@ -22,22 +21,24 @@ def test_commit_format():
     and if commits are signed.
     """
     # Fetch the upstream repository.
-    fetch_base_cmd = "git fetch {} {}".format(BASE_REPO, BASE_BRANCH)
-    subprocess.run(fetch_base_cmd, shell=True, check=True)
+    fetch_base_cmd = "git fetch -q {} {}".format(BASE_REPO, BASE_BRANCH)
+    fetch_stderr = get_cmd_result(fetch_base_cmd, result_type='stderr')
+    assert fetch_stderr == "", \
+        "Remote name should be origin. Base branch name should be master."
```

Comment:
The `should` word is not really accurate. The base branch can be a feature branch, in that case the user shouldn't change its name to master or something like that. The "origin, master" constraint is a test limitation, therefore the error message should reflect that.